deployments/charts/kuma/templates/cni-daemonset.yaml

```diff
@@ -50,9 +50,9 @@ spec:
       {{- end }}
       containers:
         - name: install-cni
-          image: {{ include "kuma.formatImage" (dict "image" .Values.cni.image "root" $) | quote }}
-          imagePullPolicy: Always
-          command: ["/install-cni.sh"]
+          image: "kuma/install-cni:11a644c7ffbdac38ca2507de8bba6166ed7e4952"
+          imagePullPolicy: Never
```

Comment:
Why `Never`? Should it be `IfNotPresent` or even configurable from `values.yaml`?

Reply:
The "Never" is still there until I implement the build script to publish the image to dockerhub.

Reply:
The "Never" is still there until I implement the build script to publish the image to dockerhub.

Reply:
@lukidzi - will this publish images on the branch as well? how will it work on PRs? can this wait for a separate PR?

Reply:
I've added the image to `docker.sh` how do I test it for this branch? Can I release from the branch?

Reply:
Technically you can when you modify circleci config, but since the change is pretty small, I'd just merge it, test on master and adjust with another PR if needed.